runtime-modules/forum/src/lib.rs

```diff
@@ -1224,6 +1258,22 @@ impl<T: Trait> Module<T> {
         Ok(())
     }
 
+    // Ensure moderator can manipulate thread.
+    fn ensure_is_thread_moderator(
```

Comment:
Nice construct, perhaps rename to `ensure_can_moderate_thread` for better semantic accuracy.